index.bs

```diff
@@ -306,6 +311,11 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     that this does not constitute [=user verification=] because [=TUP=], by definition, is not capable of [=biometric
     recognition=], nor does it involve the presentation of a shared secret such as a password or PIN.
 
+: <dfn>Resident Key</dfn>
+: <dfn>RK</dfn>
+:: A [=resident key=] is a flag representing whether the authenticator is capable of storing <a>credential private key</a> to 
+    generate the [RP]-specific <a>credential key pair</a>. 
```

Comment:
This definition does not make sense to me. A more natural way to define Resident Key would be as a key that can be looked up and used by the authenticator purely on the basis of its associated RP ID. We should seek to define behavior, not platform implementation details.

The important thing is that the Resident Key is a key, not a flag. If you want to define a flag that indicates an authenticator is capable of creating Resident Keys, then call it the Resident Key Capability Flag or something like that.